tests/ttnn/unit_tests/operations/test_moreh_norm.py

```diff
@@ -196,34 +289,103 @@ def test_moreh_norm(input_shape, p, dim_rtol_atol, keepdim, device):
 )
 @pytest.mark.parametrize("compute_kernel_options", compute_kernel_options, ids=compute_kernel_ids)
 def test_moreh_norm_compute_kernel_options(input_shape, p, dim_rtol_atol, compute_kernel_options, device):
+    """
+    Parametrized test for Tenstorrent's norm operation with various kernel compute options. Compares the output of Tenstorrent's norm
+    with PyTorch's norm for different input shapes, dimensions, norms, and compute kernel configurations.
+
+    Args:
+        input_shape (list of int): Shape of the input tensor.
+        p (float): The order of the norm.
+        dim_rtol_atol (list): List containing the dimension(s), relative tolerance, and absolute tolerance for the comparison.
+        compute_kernel_options: Configuration options for the compute kernel.
+        device: The device to run the computation on.
+
+    Raises:
+        AssertionError: If the computed outputs from Tenstorrent's implementation and PyTorch are not close.
+    """
     torch.manual_seed(2024)
     dim, rtol, atol = dim_rtol_atol
     run_moreh_norm(input_shape, p, dim, rtol, atol, device, compute_kernel_options=compute_kernel_options)
 
 
-@pytest.mark.parametrize("p", [2.0], ids=["p=2.0"])
+@pytest.mark.parametrize("p", [2.0, 2.5, -2.5])
 @pytest.mark.parametrize(
     "dim_rtol_atol",
     [
+        [[], 0.2, 0.2],
+        [None, 0.2, 0.2],
         [0, 0.1, 0.1],
         [1, 0.1, 0.1],
         [2, 0.1, 0.1],
+        [3, 0.1, 0.1],
+        [[0, 1], 0.1, 0.1],
+        [[0, 1, 2], 0.15, 0.15],
+        [[0, 1, 2, 3], 0.2, 0.2],
+        [[0, 1, 3], 0.15, 0.15],
+        [[0, 2, 3], 0.15, 0.15],
+        [[1, 2], 0.1, 0.1],
+        [[1, 2, 3], 0.15, 0.15],
+        [[1, 3], 0.1, 0.1],
+        [[2, 3], 0.1, 0.1],
```

Comment:
Similarly, this part is likely covered in other tests, so there's no need to run all test cases in the callback test.

Reply:
Although there are many cases, some of them are skipped. These are the tests that are not being skipped:
```bash
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_callback[input_shape=[2, 2, 77, 77]-N-p=2.0]
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_callback[input_shape=[2, 2, 77, 77]-C-p=2.0]
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_callback[input_shape=[2, 2, 77, 77]-NC-p=2.0]
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_backward_callback[input_shape=[2, 2, 77, 77]-N-p=2.0]
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_backward_callback[input_shape=[2, 2, 77, 77]-C-p=2.0]
PASSED tt-metal/tests/ttnn/unit_tests/operations/test_moreh_norm.py::test_moreh_norm_backward_callback[input_shape=[2, 2, 77, 77]-NC-p=2.0]
```
Should I delete the others and keep only one test for the forward callback and one for the backward callback?

Reply:
Regarding the `moreh_norm` skip, I'll need to look into the code to be sure, but the skipped tests are likely unnecessary.

Reply:
Yes, please ping me afterward.

Reply:
Updated as requested.